### PR TITLE
gnome.mutter: fixup color-device: Don't create profiles from obvious g…

### PR DESCRIPTION
**Pull request by kirillrdy**
###### Description of changes

this is for gnome branch, latest 43 updates fails to boot due to 
https://gitlab.gnome.org/GNOME/gnome-shell/-/issues/5875
applying patches from 
https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/2627
fixes the issue for me.

I am not sure if we can rely on those commit hashes from upstream MR, what happens if author rebases the branch ?

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/desktops/gnome/core/mutter/default.nix [hunk 1]
```diff
@@ -153,7 +174,7 @@ let self = stdenv.mkDerivation rec {
     libdir = "${self}/lib/mutter-11";
 
     tests = {
-      libdirExists = runCommand "mutter-libdir-exists" {} ''
+      libdirExists = runCommand "mutter-libdir-exists" { } ''
```
Comment (kirillrdy) [inline pkgs/desktops/gnome/core/mutter/default.nix:177]: opse, my editor auto formatted this, will fix

**Comment by jtojnar** [comment]
Thanks, I think the unmerged commits are fine for now (they will probably persist until GC is run on the repo.) Hopefully, the MR will be merged before we merge #182618.

**Comment by kirillrdy** [comment]
@jtojnar  Thanks,

I've also just now learned that I can get full MR patch using url like this

https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/2627.diff

Thats for the future

**Comment by jtojnar** [comment]
Yeah, though that one will not be stable when force pushed.

**Pull request merged**